### PR TITLE
Allow sets of attributes in fulltext expressions. Fixes #54.

### LINKED ISSUE #54: Allow to search set of fields using `fulltext`
Right now, `fulltext` supports a single field (`:for/example`) or the special symbol `:all` to search all fields.  This ticket tracks generalizing to allow a set of fields, like `#{:for/example :and/another}`.

### PR DESCRIPTION
**Pull request by rnewman**
Over to you, @ncalexan.

### CODE REVIEW

File: src/common/datomish/query/functions.cljc [hunk 1]
```diff
@@ -97,12 +97,32 @@
...
+        ;;   [(fulltext $ #{:foo/bar :foo/baz} "Noo") [[?x]]]
+        ;;
+        ;; which involves some tomfoolery here.
+        attr-constants (or
+                         (and (instance? Constant attr)
+                              (let [attr (:value attr)
+                                    intern (partial source/attribute-in-source (:source cc))]
+                                (when-not (= :any attr)
+                                  (cond
+                                    (set? attr)
```
Comment (ncalexan) [inline src/common/datomish/query/functions.cljc]: `sequential?`  Not sure how string we want to be here.

File: test/datomish/tofinoish_test.cljc [hunk 2]
```diff
@@ -460,3 +466,24 @@
         (is (= apricot-url "http://example.com/apricots/1"))
         (is (= apricot-title "A page about apricots."))
         (is (= apricot-excerpt "")))))))
+
+(deftest-db test-fulltext-set-attribute conn
+  (<? (d/<transact! conn tofino-schema))
+  (<? (d/<transact! conn
+                    [{:db/id             999
```
Comment (ncalexan) [inline test/datomish/tofinoish_test.cljc]: Do we hit the case where `:a` is `:db/fulltext true`, the value would match, but it's excluded from the search?  I don't think we do here.

File: src/common/datomish/query/functions.cljc (hunk 1 above)
Comment (rnewman) [inline src/common/datomish/query/functions.cljc]: I kinda like that sets enforce no ordering and no duplicates…

File: test/datomish/tofinoish_test.cljc (hunk 2 above)
Comment (rnewman) [inline test/datomish/tofinoish_test.cljc]: Added.

**Comment by rnewman** [comment]
Landed in 1296b80.

**Pull request closed without merging**